cloudify/dispatch.py

```diff
@@ -160,13 +161,17 @@ def run_subprocess(self, *subprocess_args, **subprocess_kwargs):
         subprocess_kwargs.setdefault('stderr', subprocess.STDOUT)
         subprocess_kwargs.setdefault('stdout', subprocess.PIPE)
         p = subprocess.Popen(*subprocess_args, **subprocess_kwargs)
+        if self._process_registry:
```

Comment:
as far as this class is concerned, it might well be, since it's an optional argument.
It'll be none in tests that use the OperationHandler directly, and in `cfy local` which doesnt use the worker